Yoast/wordpress-seo
# Add keyword to admin bar menu

## Summary

This PR can be summarized in the following changelog entry:

* Displays keyword in Yoast SEO admin bar menu.

## Relevant technical choices:

*

## Test instructions

This PR can be tested by following these steps:

* Install and activate Yoast SEO.
* Add a new post.
* Enter a focus keyphrase.
* Hover over the Yoast SEO menu in the admin bar.
* See the focus keyphrase in the submenu.


## UI changes

## Documentation

## Quality assurance


Fixes #8327
